CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.2.0 (2019-03-22)
+
+- Added multiselect toolbar. [#21](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/21)
```

Comment:
Aren't we "adding support for" `list-builder`'s toolbar, or am I getting that wrong?

Reply:
Yeah, plus we're also adding some properties to enable it. Maybe it should read more like `Added ability to enable multiselect toolbar` ?

Reply:
Or, "Added support for `@skyux/list-builder@^3.2.1` which includes the ability to add a multiselect toolbar to the `SkyListBuilderViewGrids` component." Something like that, @blackbaud-johnly ?

Reply:
How about "Added support for `@skyux/list-builder@^3.2.1` to add multiselect toolbar to list view grid component."? (I opted for "list view grid component" instead of "`SkyListBuilderViewGrids` component" because that is more consistent with other changelog entries.)

Reply:
Works for me!